src/mixins/filing-template-mixin.ts

```diff
@@ -463,19 +473,17 @@ export default class FilingTemplateMixin extends Mixins(AmalgamationMixin, DateM
         ...draftFiling.continuationIn.contactPoint,
         confirmEmail: draftFiling.continuationIn.contactPoint.email
       })
-    } else {
-      this.setBusinessContact({ ...EmptyContactPoint })
     }
 
     // restore Persons and Organizations
     if (draftFiling.continuationIn.parties) {
-      this.setOrgPersonList(draftFiling.continuationIn.parties || [])
+      this.setOrgPersonList(draftFiling.continuationIn.parties)
     }
 
     // restore Share Structure
-    this.setShareClasses(draftFiling.continuationIn.shareStructure
-      ? draftFiling.continuationIn.shareStructure.shareClasses
-      : [])
+    if (draftFiling.continuationIn.shareStructure?.shareClasses) {
+      this.setShareClasses(draftFiling.continuationIn.shareStructure.shareClasses)
+    }
```

Comment:
If share classes is not in the draft, we should be able to leave the initial value in the store (which is `[]`).